### DISCUSSION TITLE
Tab Group Rule Options

### DISCUSSION THREAD

**Discussion by wynnfarm** [furybee/chrome-tab-modifier#279] [Ideas]
_In case of bug report, please fill this template and remove italic lines_
_Otherwise, clear all and type :)_

### Tab Modifier version

**Browser:** `Chrome` or `Opera`
Chrome
**Extension version:** `X.Y.Z`

_Can be found here: chrome://extensions/?id=hcbgadmbdkiilgpifjgcakjehmafcjai_

### Expected behavior

_What do you expect?_
The option to have tabs added to Tab Groups 
1.  Add Rule
2. On rule details provide option to add the tab to a group
3. If group does not exist, group would be created when tab is opened
4. Option to set color of the group (optional)
5. Ability to add/Create Group based on domain name or other regex/selector options (all tabs opened in a specific domain or sub domain)
6. When the new tab is loaded it would be added to an existing group that matches the name in the rule.  

### Actual behavior

_What actually happen?_
Option not currently available
### Steps to reproduce the behavior

_Please describe the steps to help me to reproduce the actual behavior_

**Comment by cjx3711** [comment]
I second this. I've been trying to use tab groups but it's so hard to keep it organised, having this as a feature would make using tab groups in chrome so much more usable.

**Comment by kayeppes** [comment]
I agree, I actively use tab groups in Chrome and having to reassign my renamed tabs back to their respective groups after every refresh is getting enormously tedious already. Please consider this one!!! +1 million, lol.

**Comment by sylouuu** [comment]
Now available since 1.0!
